code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -456,9 +456,12 @@ var/global/list/damage_icon_parts = list()
 
 	//base icons
 	var/icon/hair_standing	= new /icon('icons/mob/human_face.dmi',"bald_s")
+	//var/icon/debrained_s = new /icon("icon"='icons/mob/human_face.dmi', "icon_state" = "debrained_s")

```

Comment:
@Aurorablade Nevermind, cached icon generation doesn't apply to this specifically. Try calling human.update_hair() in every place where a brain can be removed, it will probably work.